test/components/xDragBoxLayerTests.ts

```diff
@@ -1,23 +1,14 @@
 ///<reference path="../testReference.ts" />
 
-describe("Layer Components", () => {
+describe("SelectionBoxLayer", () => {
```

Comment:
I thought we layer describe blocks by class inheritance, if not I will change everything to `Interactive Components`